wgc/wgc_api.py

```diff
@@ -650,9 +654,11 @@ async def fetch_product_list(self) -> List[WGCOwnedApplication]:
             except:
                 logging.exception('wgc_api/fetch_product_list: failed to get app_id')
 
-            if app_gameid in self.GAMES_F2P or app_gameid in purchased_gameids:
-                is_purchased = app_gameid in purchased_gameids and app_gameid not in self.GAMES_F2P
+            if app_gameid in GAMES_F2P or app_gameid in purchased_gameids:
+                is_purchased = app_gameid in purchased_gameids and app_gameid not in GAMES_F2P
                 product_list.append(WGCOwnedApplication(product, is_purchased))
+            else:
+                logging.warning('wgc_api/fetch_product_list: unknown ID %s' % app_gameid)
```

Comment:
@Mixaill BTW you can create sub-loggers instead of root logger with custom text repeated every time

https://docs.python.org/3/library/logging.html#logger-objects